docs/spark-procedures.md

```diff
@@ -493,6 +493,38 @@ CALL spark_catalog.system.add_files(
 )
 ```
 
+### `register_table`
+
+Creates a catalog entry for a metadata.json file which already exists but does not have a corresponding catalog identifier.
+
+#### Usage
+
+| Argument Name | Required? | Type | Description |
+|---------------|-----------|------|-------------|
+| `table`       | ✔️  | string | Table which is to be registered |
+| `metadata_file`| ✔️  | string | Metadata file which is to be registered as a new catalog identifier |
+
+!!! Warning
+Having the same metadata.json registered in more than one catalog can lead to missing updates, loss of data, and table corruption. Only use this procedure when the table is no longer registered in an existing catalog, or you are moving a table between catalogs.
```

Comment:
This line needs to be indented to have the block work

Please check all do changes after rendering the page to make sure it works

To do so follow the directions here
https://github.com/apache/iceberg-docs/tree/main

Once I copied the files into the iceberg-docs repo I ran
hugo in the docs directory to generate the html in the public dir

![image](https://user-images.githubusercontent.com/413025/206635505-1e59295c-97d3-4fb8-810b-79e50089f6a7.png)

Reply:
Or you can run the server and the page will be at
http://localhost:1313/spark-procedures/

Reply:
I tried indenting and running the server, but `!!! Warning` didn't work. Also, I don't find any occurrences of `!!! Warning` in the code base.

Reply:
Hint warning is used in the Iceberg hive docs: https://github.com/apache/iceberg/blob/master/docs/hive.md. It displays a yellow block: https://iceberg.apache.org/docs/latest/hive/

Reply:
Here is how it looks with `{{< hint warning >}} ... {{< /hint >}}`:

<img width="828" alt="Screenshot 2022-12-09 at 2 41 13 PM" src="https://user-images.githubusercontent.com/4561678/206666389-f7eb2be3-c163-4a1b-a251-b99f026e48b9.png">

Reply:
I think even `!!! note` is being replaced with `{{< hint info >}} ... {{< /hint >}}` in the codebase. I see few occurrences here: https://github.com/apache/iceberg/blob/master/docs/spark-writes.md#writing-to-partitioned-tables

Reply:
Ah, I must be behind the times. Thanks for checking!